elide-datastore/elide-datastore-jpa/src/test/java/com/yahoo/elide/datastores/jpa/JpaDataStoreHarness.java

```diff
@@ -96,7 +99,8 @@ public JpaDataStoreHarness() {
 
         store = new JpaDataStore(
                 () -> { return emf.createEntityManager(); },
-                (entityManager) -> { return new NonJtaTransaction(entityManager); }
+                (entityManager, txCancel) -> { return new NonJtaTransaction(entityManager, txCancel); },
```

Comment:
So for example, there is no need to include txCancel as a parameter to the supplier.  Rewrite like:
```java
(entityManager) -> { return new NonJtaTransaction(entityManager, jpaTransactionCancel); }
```